gtk4/src/auto/application.rs

```diff
@@ -22,6 +23,101 @@ glib::wrapper! {
     }
 }
 
+impl Application {
+    #[doc(alias = "gtk_application_new")]
+    pub fn new(application_id: Option<&str>, flags: gio::ApplicationFlags) -> Application {
+        assert_initialized_main_thread!();
```

Comment:
This should also call `register_startup_hook()` or not?

Reply:
I don't think so because [`gtk_application_new`](https://developer.gnome.org/gtk3/stable/GtkApplication.html#gtk-application-new) should do everything for us as of docs.

Reply:
It does not call https://github.com/gtk-rs/gtk4-rs/blob/8107dfccc9c8a3939f1eaf45b23911dfc05d986e/gtk4/src/rt.rs#L69 though

Reply:
This sounds like a problem :upside_down_face:

Reply:
So we want to manual version but should no longer return a `Result`

Reply:
I think so. This is all a bit annoying :)